packages/sit-onyx/src/styles/density.scss

```diff
@@ -0,0 +1,61 @@
+$density-compact: 2rem;
+$density-default: 2.5rem;
+$density-cozy: 3rem;
+
+@mixin compact {
+  .onyx-density-compact &,
+  :is(&).onyx-density-compact {
+    @content;
+  }
+}
+
+@mixin default {
+  &,
+  .onyx-density-default &,
+  :is(&).onyx-density-default {
+    @content;
+  }
+}
+
+@mixin cozy {
+  .onyx-density-cozy &,
+  :is(&).onyx-density-cozy {
+    @content;
+  }
+}
+
+.onyx-density-compact {
+  --onyx-density: #{$density-compact};
+}
+
+:root,
+.onyx-density-default {
+  --onyx-density: #{$density-default};
+}
+
+.onyx-density-cozy {
+  --onyx-density: #{$density-cozy};
+}
+
+@media (pointer: fine) {
+  .onyx-density-fine-compact {
+    --onyx-density: #{$density-compact};
+  }
+  .onyx-density-fine-default {
+    --onyx-density: #{$density-default};
+  }
+  .onyx-density-fine-cozy {
+    --onyx-density: #{$density-cozy};
+  }
+}
+@media (pointer: coarse) {
+  .onyx-density-coarse-compact {
+    --onyx-density: #{$density-compact};
+  }
+  .onyx-density-coarse-default {
+    --onyx-density: #{$density-default};
+  }
+  .onyx-density-coarse-cozy {
+    --onyx-density: #{$density-cozy};
+  }
+}
```

Comment:
Isn't this the same as the regular density?

Reply:
Not sure if I understood you correctly. But we offer each option (even the defaults), so that they can be overwritten for any subtree.